packages/eslint-plugin/src/rules/unbound-method.ts

```diff
@@ -18,6 +18,59 @@ export type Options = [Config];
 
 export type MessageIds = 'unbound';
 
+const nativelyBoundMembers = ([
+  'Promise',
```

Comment:
I think this is wrong, static `Promise` methods are not bound but rely on their `this` value (to use as a constructor). I think `Array` is similar but I could be wrong.

Reply:
You've misunderstood the point of this change.
This change doesn't allow you to do `const unbound = promiseVar.then`
It allows you to use the static methods on these names `const nativelyBound = Math.abs`.

In this case, you happen to be partially correct - the Promise statics are actually not bound, and using them not on the `Promise` object will throw an error.

These are the exceptions to this list:
```
Promise.all
Promise.race
Promise.resolve
Promise.reject
Promise.allSettled
Object.defineProperties
Object.defineProperty
Reflect.defineProperty
Reflect.deleteProperty
Reflect.get
Reflect.getOwnPropertyDescriptor
Reflect.getPrototypeOf
Reflect.has
Reflect.isExtensible
Reflect.ownKeys
Reflect.preventExtensions
Reflect.set
Reflect.setPrototypeOf
```

Reply:
No, I understood this. I wanted to complain that `Promise.all`, `Promise.race`, `Promise.allSettled`, `Promise.resolve` and `Promise.reject` **should not have become exceptions** - they rely on being called with `Promise` (or a compatible subclass) as their `this` value.